version/version.go

```diff
@@ -16,5 +16,5 @@ package version
 
 var (
 	// Version is the ArgoCD Operator version.
-	Version = "0.0.16"
+	Version = "0.1.0"
```

Comment:
Please get rid of this version file/package.   Instead, we can set the -X in -ldflags and use VERSION variable in Makefile.  Please follow the pattern e.g. in kam.
https://github.com/redhat-developer/kam/blob/master/Makefile#L10
to set the Version variable in go code.
https://github.com/redhat-developer/kam/blob/master/pkg/cmd/version/version.go#L13

Reply:
Done